tools/azsdk-cli/Azure.Sdk.Tools.Cli/Program.cs

```diff
@@ -30,4 +20,61 @@ public static async Task<int> Main(string[] args)
 
         return await parsedCommands.InvokeAsync(args);
     }
+
+    public static bool IsCLI(string[] args) => !args.Select(x => x.Trim().ToLowerInvariant()).Any(x => x == "start");
```

Comment:
Can we put a TODO here to make this a bit more intelligent (e.g. excluding start subcommands)?

Reply:
Yeah.